src/app/utils/plug-descriptions.ts

```diff
@@ -152,6 +153,12 @@ function getPerkDescriptions(
           perkDescription = undefined;
         } else {
           usedStrings.add(perkDescription);
+          results.push({
```

Comment:
Latest push reverts the changes in plug-descriptions. I originally separated the description and requirement pushes because the garden of salvation mods had description and requirements together and one of my early attempts at a fix needed them to be apart.  I'm realizing the fix I have now doesn't need those changes.